### ISSUE TITLE
Domain Failure - Reset Stuck at SSL

### ISSUE THREAD

**Issue by comradepanda** [aws-amplify/amplify-hosting#3764]
### Before opening, please confirm:

- [X] I have checked the [custom domains troubleshooting guide](https://docs.aws.amazon.com/amplify/latest/userguide/custom-domain-troubleshoot-guide.html).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-hosting/blob/master/CONTRIBUTING.md).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### App Id

d1e3hs66s1g71g

### Region

us-west-2

### Step

SSL Configuration 

### Status

Stuck

### Issue or question

I had a custom domain name operational for some time: https://dreamwish.studio and the DNS stopped resolving to the website. 

amplifyapp.com link is still functional. 

Decided best fix was delete and reconfigure the custom domain settings. 

Recreation is stuck at verifying SSL. https://www.whatsmydns.net/#CNAME/www.dreamwish.studio shows no active results 
![image](https://github.com/aws-amplify/amplify-hosting/assets/6163591/1b1e6435-1e69-4efe-b7b9-f18990ad4d55)

I tried updating the NS record to suggested servers from the guide.

ns-1192.awsdns-21.org.
ns-294.awsdns-36.com.
ns-1886.awsdns-43.co.uk.
ns-953.awsdns-55.net.

Domain is hosted in route 53 and records are present for verification as well as CNAMEs. 

Still stuck currently. 

### Error message

Stuck at SSL verification. 

### Additional information

TL:DR

1. Custom domain was functional. Then because NX no record. 
2. Deleted custom domain configuration. 
3. Re-implemented custom domain configuration. 
4. Stuck at SSL verification. 
5. Updated AWS NS record to match servers from the trouble shooting guide. 
6. Made this ticket.

**Comment by Jay2113** [comment]
Hi @comradepanda 👋 , thanks for reaching out. I observed that have you successfully re-activated the dreamwish.studio domain on your Amplify app `d1e3hs66s1g71g` and it seems to be resolving as expected.

If the issue re-surfaces, I'll recommend reviewing the DNS settings in Route53 for the following CNAME records:
- CNAME pointing to CloudFront distribution for root and sub domains
- CNAME pointing to an ACM server validation record

These records are required within the DNS settings at the time of domain activation and even post activation. For more information related to troubleshooting custom domain issues, you can refer to this guide: https://docs.aws.amazon.com/amplify/latest/userguide/custom-domain-troubleshoot-guide.html.
